include/deal.II/grid/reference_cell.h

```diff
@@ -364,6 +364,13 @@ class ReferenceCell
   std_cxx20::ranges::iota_view<unsigned int, unsigned int>
   face_indices() const;
 
+  /**
+   * Return an object that can be thought of as an array containing the indices
+   * of the givent cell's faces of the desired type @p face_ref_type.
+   */
+  std_cxx20::ranges::iota_view<unsigned int, unsigned int>
+  face_indices_by_type(ReferenceCell face_ref_type) const;
```

Comment:
The return type makes the assumption that these faces form a contiguous range of indices. This may be true, but it's at least not obvious. I think it might be worth saying so. You might also want to mention that that range just happens to be empty if no such face exists.

Separately:
```suggestion
  /**
   * Return an object that can be thought of as an array containing the indices
   * of the givent cell's faces of the desired type @p face_ref_type.
   */
  std_cxx20::ranges::iota_view<unsigned int, unsigned int>
  face_indices_by_type(const ReferenceCell &face_ref_type) const;
```

Reply:
Done.